src/publishProject/formComponentHelpers.ts

```diff
@@ -13,44 +13,39 @@ import {
     PublishTarget,
     PublishFormFields,
 } from "../sharedInterfaces/publishDialog";
-import {
-    getPublishServerName,
-    validateSqlServerPortNumber,
-    isPreviewFeaturesEnabled,
-} from "./projectUtils";
+import { getPublishServerName, validateSqlServerPortNumber } from "./projectUtils";
 import { validateSqlServerPassword } from "../deployment/dockerUtils";
 
 /**
  * Generate publish target options based on project target version
  * @param projectTargetVersion - The target version of the project (e.g., "AzureV12" for Azure SQL)
  * @returns Array of publish target options
  */
-function generatePublishTargetOptions(projectTargetVersion?: string): FormItemOptions[] {
-    // Check if this is an Azure SQL project
-    const isAzureSqlProject = projectTargetVersion === constants.AzureSqlV12;
+function generatePublishTargetOptions(): FormItemOptions[] {
     const options: FormItemOptions[] = [
         {
-            displayName: isAzureSqlProject
-                ? Loc.PublishTargetExistingLogical
-                : Loc.PublishTargetExisting,
+            displayName: Loc.PublishTargetExisting,
             value: PublishTarget.ExistingServer,
         },
         {
-            displayName: isAzureSqlProject
-                ? Loc.PublishTargetAzureEmulator
-                : Loc.PublishTargetContainer,
+            displayName: Loc.PublishTargetContainer,
             value: PublishTarget.LocalContainer,
         },
     ];
-    if (isAzureSqlProject) {
-        // Only show "Publish to New Azure Server" option if preview feature tag is enabled
-        if (isPreviewFeaturesEnabled()) {
-            options.push({
-                displayName: Loc.PublishTargetNewAzureServer,
-                value: PublishTarget.NewAzureServer,
-            });
+    // TODO: Hiding the logical contianer publishing target option till we provide the full publishing experience
+    /*
+    // Check if this is an Azure SQL project
+        const isAzureSqlProject = projectTargetVersion === constants.AzureSqlV12;
+        if (isAzureSqlProject) {
+            // Only show "Publish to New Azure Server" option if preview feature tag is enabled
+            if (isPreviewFeaturesEnabled()) {
+                options.push({
+                    displayName: Loc.PublishTargetNewAzureServer,
+                    value: PublishTarget.NewAzureServer,
+                });
+            }
         }
-    }
+    */
```

Comment:
We can just remove this entirely; we don't want commented-out dead code checked in, even if it's something we're planning to add soon.